src/Core/Form/ErrorMessage/AdministrationConfigurationError.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Form\ErrorMessage;
+
+/** Configuration error specific to administration page */
+class AdministrationConfigurationError implements ConfigurationErrorInterface
+{
+    public const ERROR_COOKIE_LIFETIME_MAX_VALUE_EXCEEDED = 1;
+    public const ERROR_COOKIE_SAMESITE_NONE = 2;
+
+    /**
+     * @var int
+     */
+    private $errorCode;
+
+    /**
+     * @var string
+     */
+    private $fieldName;
+
+    /**
+     * @var int|null
+     */
+    private $languageId;
+
+    /**
+     * AdministrationConfigurationError constructor.
+     *
+     * @param int $errorCode
+     * @param string $fieldName
+     * @param int|null $languageId
+     */
+    public function __construct(int $errorCode, string $fieldName, ?int $languageId = null)
+    {
+        $this->errorCode = $errorCode;
+        $this->fieldName = $fieldName;
+        $this->languageId = $languageId;
+    }
+
+    /**
+     * @return int
+     */
+    public function getErrorCode(): int
+    {
+        return $this->errorCode;
+    }
+
+    /**
+     * @return string
+     */
+    public function getFieldName(): string
+    {
+        return $this->fieldName;
+    }
+
+    /**
+     * @return int|null
+     */
+    public function getLanguageId(): ?int
```

Comment:
Where is this function used ?

Reply:
It's needed in the future, but after thinking about it yeah it's probably not needed now. Removed it.